src/encoded/tests/test_create_mapping.py

```diff
@@ -1,10 +1,14 @@
 import pytest
+
+from snovault import COLLECTIONS
 from unittest.mock import patch, MagicMock
+from ..commands.create_mapping_on_deploy import ITEM_INDEX_ORDER
+from ..commands.create_mapping_on_deploy import get_deployment_config
+# TODO: We should not be importing *. Even stranger, PyCharm says we don't use anything from there. -kmp 14-Feb-2020
```

Comment:
Does removing the below import have any side effects? Also there are still imports in `test_create_mapping` below if you want to pull those up.

Reply:
I didn't try removing it. But I think PyCharm code walks it. Variables would light up red as having no definition if it created a problem. Instead, it makes this gray, indicating it thinks it's unused. (What it doesn't codewalk is whether anyone imports anything from this file that might have been an apparently-unused import.) I put the comment as a note to remind myself to look more at this later.